File: backend/src/main/java/io/kongkham/kcurr/ApiClients/CurrencyBeacon/CurrencyBeaconCurrCountriesApiResponse.java
package io.kongkham.kcurr.ApiClients.CurrencyBeacon;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
public class CurrencyBeaconCurrCountriesApiResponse {
    @JsonProperty("response")
    private CurrencyBeaconCurrCountriesApiResponseResponse[] response ;
}

File: backend/backend/ApiClients/CurrencyBeacon/CurrencyBeaconCurrCountriesApiResponse.cs
using System.Text.Json.Serialization;

namespace backend.ApiClients.CurrencyBeacon;

public class CurrencyBeaconCurrCountriesApiResponse
{
    [JsonPropertyName("response")]
    public CurrencyBeaconCurrCountriesApiResponseResponse[] Response { set; get; }
}